src/containers/cards/actions.js

```diff
@@ -56,3 +56,18 @@ export const saveCardRank = card => (dispatch) => {
     },
   })
 }
+
+export const saveCardTitle = (cardId, cardTitle) => (dispatch) => {
+  dispatch({
+    type: RENAME_CARD,
```

Comment:
It's expected to have the action method with the same name that the action constant. Choose between `saveCardTitle` and `renameCard`